components/ClaimPage/ClaimPage.tsx

```diff
@@ -1,46 +1,147 @@
-import { useEffect, useState } from "react"
+import { useMeasure } from "react-use"
+import { useRef } from "react"
+import { useMediaQuery, useReadLocalStorage } from "usehooks-ts"
 import { useSigner } from "wagmi"
-import { BigNumber } from "ethers"
-import abi from "../../lib/abi-cre8ors.json"
-import NavBar from "../NavBar"
-import balanceOf from "../../lib/balanceOf"
-import ClaimButton from "../ClaimButton"
-import MintButton from "../MintButton"
+import Layout from "../Layout"
+import SectionTitle from "../LandingPage/SectionTitle"
+import SectionContent from "../LandingPage/SectionContent"
+import { Button } from "../../shared/Button"
+import Media from "../../shared/Media"
+import Footer from "../Footer"
+import { useTheme } from "../../providers/ThemeProvider"
+import useGradualFadeEffect from "../../hooks/useGradualFade"
+import Popover from "../../shared/Popover"
+import ConnectWallet from "./ConnetWallet"
+import Redeem from "./Redeem"
 
-function ClaimPage() {
+const ClaimPage = () => {
```

Comment:
same feedback as Claim Page

Why is Claim Page code in your pull request for Roadmap Page?

blocker: please remove Claim Page code from your Roadmap Page pull request 🚮